src/library/coverartdelegate.cpp

```diff
@@ -63,11 +62,11 @@ void CoverArtDelegate::slotInhibitLazyLoading(
     }
     // If we can request non-cache covers now, request updates
     // for all rows that were cache misses since the last time.
-    auto staleRows = m_cacheMissRows;
     // Reset the member variable before mutating the aggregated
     // rows list (-> implicit sharing) and emitting a signal that
     // in turn may trigger new signals for CoverArtDelegate!
-    m_cacheMissRows = QList<int>();
+    QList<int> staleRows = std::move(m_cacheMissRows);
```

Comment:
I think we can get rid of intermediate staleRows. I am unsure if the compiler is smart enough to optimize it away.

Reply:
Sure. But I wanted to keep the debug assertion, just in case.

Reply:
It can be moved below emitRowsChanged()